src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -784,6 +786,9 @@ enablePact48 v cid bh = [FlagDisablePact48 | not (chainweb220Pact v cid bh)]
 enablePact49 :: ChainwebVersion -> V.ChainId -> BlockHeight -> [ExecutionFlag]
 enablePact49 v cid bh = [FlagDisablePact49 | not (chainweb221Pact v cid bh)]
 
+enablePact410 :: ChainwebVersion -> V.ChainId -> BlockHeight -> [ExecutionFlag]
+enablePact410 v cid bh = [FlagDisablePact410 | not (chainweb222Pact v cid bh)]
+
 -- | Even though this is not forking, abstracting for future shutoffs
 disableReturnRTC :: ChainwebVersion -> V.ChainId -> BlockHeight -> [ExecutionFlag]
 disableReturnRTC _v _cid _bh = [FlagDisableRuntimeReturnTypeChecking]
```

Comment:
this needs to similarly hook up the webauthn signature validation for `applyLocal`, no?